e2e/pyrightconfig.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Just flagging this in case it was added by mistake.

Reply:
It's not a mistake, was actually needed to get `pyright` to work properly on my machine, so figure I would leave it here as it may help others as well.